shipyard-controller/db/mongodb_project_credentials_repo.go

```diff
@@ -0,0 +1,101 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	apimodels "github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/keptn/shipyard-controller/models"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type MongoDBProjectCredentialsRepo interface {
```

Comment:
please add a short explanation as a comment to describe why this struct is needed in addition to the previous projectsMVRepo - also, just to clarify, this will be removed in the next version when we don't need to migrate anymore, right?

Reply:
added and yes, exactly :)